src/main/webapp/components/BuildSnapshot.js

```diff
@@ -1,14 +1,72 @@
-import * as React from "react";
+import * as React from 'react';
+import {getFields} from './utils/getFields.js';
 
 /**
  * Component used to render Build Information.
  *
  * @param {*} props input properties containing build information
  */
 export const BuildSnapshot = (props) => {
+  // State variable indicating whether the BuildSnapshot is open or closed.
+  // isOpen === true indicates the tray should be visible 
+  // and the indicator arrow on the Header should be facing downwards.
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const headerFields = ['description', 'commitHash', 'repository', 'status'];
+  const trayFields = ['builders', 'timestamp'];
+
+  const headerData = getFields(props.buildData, headerFields, '');
+  const trayData = getFields(props.buildData, trayFields, '');
+
+  return (
+    <div className = 'build-snapshot'>
```

Comment:
please remove the spaces around `=` here and elsewhere; that's consistent with current code, e.g., https://github.com/googleinterns/step240-2020/blob/8f51a5114d68bb764efad1c8cb6c6b1cd6143a88/src/main/webapp/components/BuildSnapshotContainer.js#L27

Reply:
Whitespace removed here bc2b087